webapi/appsettings.Development.json

```diff
@@ -0,0 +1 @@
+{ "AIService": { "Type": "OpenAI", "Endpoint": "", "Models": { "Completion": "gpt-3.5-turbo", "Embedding": "text-embedding-ada-002", "Planner": "gpt-3.5-turbo" } }, "Authentication": { "Type": "None", "AzureAd": { "Instance": "", "TenantId": "", "ClientId": "", "Scopes": "access_as_user" } } }
```

Comment:
Did you mean to check this in?

Reply:
I don't need to but I saw no sensitive info, so I committed it.

Reply:
There is no sensitive info, but the file doesn't need to be added either.

Reply:
I will remove it in a following commit.